save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/ParseUrlInTab.kt

```diff
@@ -46,23 +56,29 @@ fun <T : Enum<T>, S : HasSelectedMenu<T>> AbstractView<*, S>.urlAnalysis(menu: T
     }
 }
 
+/**
+ * @param pathDefaultTab
+ * @param suffix
+ */
+fun <T : Enum<T>>NavigateFunctionContext.generateLinksWithSuffix(pathDefaultTab: String, suffix: String) {
```

Comment:
```suggestion
fun <T : Enum<T>>NavigateFunctionContext.navigateToLinkWithSuffix(pathDefaultTab: String, suffix: String) {
```
This doesn't generate anything, this performs navigation

Reply:
Done